scilpy/tractograms/tractogram_operations.py

```diff
@@ -879,6 +879,44 @@ def split_sft_randomly_per_cluster(orig_sft, chunk_sizes, seed, thresholds):
     return final_sfts
 
 
+def filter_tractogram_data(tractogram, streamline_ids):
+    """Filter tractogram according to streamline ids and keep the data
+
+    Parameters:
+    -----------
+    tractogram: StatefulTractogram
+        Tractogram containing the data to be filtered
```

Comment:
I think we finish description with dots. But not sure and I don't know our convention for this.